### PR TITLE
Moved setup directions

### PR DESCRIPTION
**Pull request by Jose-27**
Moved setup directions to main repo docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,121 +12,7 @@ Exoframe is a self-hosted tool that allows simple one-command deployments using
 
 ## Installation and Usage
```
Comment (yamalight) [inline README.md]: Rename this section to "Installation, usage and docs"

File: README.md [hunk 2]
```diff
@@ -12,121 +12,7 @@ Exoframe is a self-hosted tool that allows simple one-command deployments using
...
-publicKeysPath: '/path/to/your/public/keys'
-
-# whether Exoframe server whould be running in swarm mode, default "false"
-swarm: false
-```
-
-_Warning:_ Most changes to config are applied immediately. With exception of Letsencrypt config. If you are enabling letsencrypt after Traefik instance has been started, you'll need to remove Traefik and then restart Exoframe server for changes to take effect.
+For more details on how to get up an runing please falow the link [ How to setup exoframe-server](https://github.com/exoframejs/exoframe/tree/master/docs).
 
 ## Docs
```
Comment (yamalight) [inline README.md]: Kill the docs section as it essentially doubles the same link now 🤔

File: README.md [hunk 3]
```diff
@@ -12,121 +12,7 @@ Exoframe is a self-hosted tool that allows simple one-command deployments using
...
-
-# path to folder with authorized_keys, default "~/.ssh"
-publicKeysPath: '/path/to/your/public/keys'
-
-# whether Exoframe server whould be running in swarm mode, default "false"
-swarm: false
-```
-
-_Warning:_ Most changes to config are applied immediately. With exception of Letsencrypt config. If you are enabling letsencrypt after Traefik instance has been started, you'll need to remove Traefik and then restart Exoframe server for changes to take effect.
+For more details on how to get up an runing please falow the link [ How to setup exoframe-server](https://github.com/exoframejs/exoframe/tree/master/docs).
```
Comment (yamalight) [inline README.md]: Please, proof read that line a bit, there's a couple of typos here as well as some unneeded capitalization and extra spaces :)

**Comment by Jose-27** [comment]
all fixed!

**Comment by yamalight** [comment]
@Jose-27 looks good, thank you! 👍

**Pull request merged**